### ISSUE TITLE
Typo on deactivation email?

### ISSUE THREAD

**Issue by mheadd** [cloud-gov/uaa-bot#5]
Our email notification on deactivating currently advises users to `please email support@cloud.gov within the last 90 days`. Should [this](https://github.com/cloud-gov/uaa-bot/blob/main/uaa_bot/templates/account_expired.html#L14) say `please email support@cloud.gov within the next 90 days`? Current wording doesn't seem to make sense.

**Comment by mheadd** [comment]
Thoughts, @apburnes?

**Comment by apburnes** [comment]
Good catch.  Honestly, the emailing support and 90 days aren't really related so I removed the `90 days` from the emailing support statement.  Let me know what you think. https://github.com/cloud-gov/uaa-bot/pull/6
